docs/usage.md

```diff
@@ -181,123 +249,236 @@ For example:
+
+More information in the [sentieon](sentieon.md) documentation.
+
+### --skip_qc
+
+Use this to disable specific QC and Reporting tools.
+Available: `all`, `bamQC`, `BaseRecalibrator`, `BCFtools`, `Documentation`, `FastQC`, `MultiQC`, `samtools`, `vcftools`, `versions`
+Default: `None`
+
+### --target_bed
+
+Use this to specify the target BED file for targeted or whole exome sequencing.
+
+### --tools
+
+Use this parameter to specify the variant calling and annotation tools to be used.
+Multiple tools can be specified.
 For example:
 
 ```bash
---sample sample.tsv
+--tools 'Strelka,mutect2,SnpEff'
 ```
 
-Multiple TSV files can be specified if the path must be enclosed in quotes
+Available variant callers: `ASCAT`, `ControlFREEC`, `FreeBayes`, `HaplotypeCaller`, `Manta`, `mpileup`, `MSIsensor`, `Mutect2`, `Strelka`, `TIDDIT`.
+
+> `/!\` Not all variant callers are available for both germline and somatic variant calling.
+For more details please check the [variant calling](variant_calling.md) extra documentation.
+
+Available annotation tools: `VEP`, `SnpEff`, `merge`. For more details, please check the [annotation](annotation.md) extra documentation.
+
+## Modify fastqs (trim/split)
+
+### --trim_fastq
+
+Use this to perform adapter trimming [Trim Galore](https://github.com/FelixKrueger/TrimGalore/blob/master/Docs/Trim_Galore_User_Guide.md)
+
+### --clip_r1
+
+Instructs Trim Galore to remove a number of bp from the 5' end of read 1 (or single-end reads).
+This may be useful if the qualities were very poor, or if there is some sort of unwanted bias at the 5' end.
+
+### --clip_r2
+
+Instructs Trim Galore to remove a number of bp from the 5' end of read 2 (paired-end reads only).
+This may be useful if the qualities were very poor, or if there is some sort of unwanted bias at the 5' end.
+
+### --three_prime_clip_r1
 
-Use this to specify the location to a directory on `mapping` step with a single germline sample only.
+Instructs Trim Galore to remove a number of bp from the 3' end of read 1 (or single-end reads) AFTER adapter/quality trimming has been performed.
+This may remove some unwanted bias from the 3' end that is not directly related to adapter sequence or basecall quality.
+
+### --three_prime_clip_r2
+
+Instructs Trim Galore to remove a number of bp from the 3' end of read 2 AFTER adapter/quality trimming has been performed.
+This may remove some unwanted bias from the 3' end that is not directly related to adapter sequence or basecall quality.
+
+### --trim_nextseq
+
+This enables the option `--nextseq-trim=3'CUTOFF` within `Cutadapt`, which will set a quality cutoff (that is normally given with `-q` instead), but qualities of G bases are ignored.
+This trimming is in common for the NextSeq and NovaSeq-platforms, where basecalls without any signal are called as high-quality G bases.
+
+### --save_trimmed
+
+Option to keep trimmed FASTQs
+
+### --split_fastq
+
```

Comment:
Maybe mention of brief purpose of this, even though you've given a link.

Reply:
I think that the name and the description is enough.
Not sure that many people actually use that, but could be useful if you want to speed things up, or if you have a really big fastq as an input